packages/theming/atlas/src/themesource/atlas_core/web/core/widgets/_data-grid.scss

```diff
@@ -5,6 +5,10 @@
 // To customize any core styling, copy the part you want to customize to styles/web/sass/app/ so the core styling is overwritten.
 //
 
+@import "../../variables";
+@import "../../../../../theme/web/custom-variables";
+@import "../base/mixins/animations";
```

Comment:
I see this imports the mixin for animations but the mixin is _always_ imported in `main.scss`. I prefer this approach, but applied uniformly across all mixins and all files using them.

Reply:
https://github.com/mendix/widgets-resources/pull/507#discussion_r591589176
> The `meta.load-css` mixin treats the the referenced css as a module, so variables aren't available.